API/controllers/comment-controller.js

```diff
@@ -16,3 +16,20 @@ exports.getComments = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const comment = req.body;
+  return fetchArticle(article_id)
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article Not Found" });
```

Comment:
It's best to reject these promises in the models, because if you had done so in `fetchArticle`, you wouldn't need to add this promise reject here, it would already have happened in the model.
This would also be a great use case for a `Promise.all` as you're relying on both promises succeeding